include/bout/paralleltransform.hxx

```diff
@@ -137,6 +137,20 @@ private:
   /// Cache of phase shifts for transforming from field-aligned coordinates to X-Z orthogonal coordinates
   arr3Dvec fromAlignedPhs;
 
+  /// Helper POD for parallel slice phase shifts
+  struct ParallelSlicePhase {
+    arr3Dvec phase_shift;
+    int y_offset;
+  };
+
+  /// Cache of phase shifts for the parallel slices. Slices are stored
+  /// in the following order:
```

Comment:
Presumably the order shouldn't really matter as the struct carries the index with it etc. ? I don't suppose it really matters but we might (potentially but probably not) get better memory access etc. if this was ordered as `{-n, ...-1, 1, ...., n}`

Reply:
It should only matter for the initialisation, but is otherwise arbitrary. The current scheme was just easy to implement.

I suspect memory access is not the bottleneck here, but I don't have any data to support that.